src/diffusers/schedulers/scheduling_lms_discrete.py

```diff
@@ -124,10 +124,9 @@ def step(
         return {"prev_sample": prev_sample}
 
     def add_noise(self, original_samples, noise, timesteps):
-        alpha_prod = self.alphas_cumprod[timesteps]
-        alpha_prod = self.match_shape(alpha_prod, original_samples)
+        sigmas = self.match_shape(self.sigmas[timesteps], noise)
```

Comment:
We should give the original author credit here and instead merge: https://github.com/huggingface/diffusers/pull/253

Reply:
Of course, the plan is to merge the original PR before this :)